javascript/phone-number.js

```diff
@@ -0,0 +1,27 @@
+module.exports = PhoneNumber;
+
+function PhoneNumber (num) {
+  num = num.replace(/[(]|[)]|[-]|[\s]|[.]/g, '');
+  this.number = function () {
+    if (num.length === 11 && num[0] === '1') {
+      num = num.slice(1);
+      return num;
+    }
+    else if (num.length !== 10){
+      num = '0000000000';
+      return num;
+    } else {
```

Comment:
:+1: Nice job on the short `if/else` conditional. You could potentially make it shorter by eliminating this last `else` by having your method always return `num`, with your conditional modifying `num` when necessary.